scripts/set_deployment_project_dir.py

```
@@ -0,0 +1,103 @@
+else:
+    payload = {}
+
+r = requests.post(f"{BASE_URL}/deployments/filter", json=payload, timeout=20)
+r.raise_for_status()
+deployments = r.json()
+
+if args.show:
+    for deployment in deployments:
+        if deployment.get("parameters"):
+            if deployment["parameters"].get("config"):
+                if deployment["parameters"]["config"].get("tasks"):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)